talent/job-descriptions/applications.md

```diff
@@ -0,0 +1,63 @@
+- src-d/engine:  a library for running scalable data retrieval pipelines that process any number of Git repositories for source code analysis.  
+- src-d/hercules:  analysis and visualization of a Git repository history. 
+- src-d/gemini: a duplicate finder for source code in git repositories.
+
+## ROLE
+
+- This position is for a Senior Engineer in the Applications team.
+
+- As an Applications team engineer, you will have a two-fold role: on one hand it's a Data Engineering, building production-level pipelines to collect and extract data from all git repositories in the world. On the another hand it's applied Machine Learning, by productionizing Machine Learning research and wrapping it in useful end-user applications.
+
+- You will be expected to have strong backend coding skills in at least one programming language and very good algorithmic skills. Scala coding skills and knowledge about Apache Spark aren't required but will be highly appreciated.
+
+- Building tools for day to day use by software engineers require good understanding of modern development processes and approaches: Unit/Integration testing, scrum, code reviews, CI/CD, Containers, etc.
+
+- As you will be working closely with the Machine Learning team - strong communication skills and interest in Machine Learning, data process, and visualization are more than welcome. 
+
+- You will deal with models built using modern ML stacks written in Python and C++ like Tensorflow, Tensorboard, CUDA, so familiarity with any of those tools is a plus.
+
+## CULTURE
+
+- source{d} is a company for developers by developers. We firmly believe in always doing what's best for the community, not necessarily companies. "For developers" is the answer to any strategic question or dilemma in the company. Always For Developers.
+
+- "By developers" is our identity. Everyone in the company is either a developer or has a developer/engineering mindset. We don't hire anyone who doesn't fit into this culture, it's the only way to make sure everyone understands developers and can always defend their interest.
+
+- At the moment, we are 20+ people from 10 different countries working closely together from our office in Madrid, Spain. We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely as part of our team does working from Portugal, Estonia, Russia, and others.
+
+- At source{d}, we have a transparent salary policy. Depending on your experience, you can either join us as a Senior Engineer or a Lead. For a Senior Engineer, the salary ranges from €49.000 to €53.000. You can find our net amounts at https://cincodias.elpais.com/herramientas/calculadora-sueldo-neto/ (in Spanish only).
```

Comment:
As this is a description of Senior Engineer position in the Applications team, the sentence

> Depending on your experience, you can either join us as a Senior Engineer or a Lead. 

 looks a bit confusing and may need to be removed.

Reply:
I removed it; on Lever it was already updated with "At source{d}, we have a transparent salary policy which we feel strongly about it. Although a seniority level for every candidate will be determined during the last round of on-site interviews, one can expect that for a Senior Engineer salary ranges from €49.000 to €53.000.". I guess I forgot to change it here.